examples/utils/get-tinylicious-container/src/getTinyliciousContainer.ts

```diff
@@ -42,6 +42,8 @@ class InsecureTinyliciousUrlResolver implements IUrlResolver {
             },
             tokens: { jwt: this.auth(documentId) },
             type: "fluid",
+            // This is wrong as it should contain the full path, but it doesn't matter
```

Comment:
I think its worth adding the rest of the path to the documentUrl since its a simple fix and is low risk. The comments say that it serves URL of this format - documentId/containerRelativePathing. So we should add the containerReltiavePathing to the resolved URL.

Reply:
If we decide not to fix it, I would suggest removing this new comment and changing the outer comment to say we only support url of format - documentId (remove containerRelativePathing).

Reply:
I believe we need to support `documentId/containerRelativePathing` such that we can successfully roundtrip through `getAbsoluteUrl()`.  It's probably fine to add containerRelativePathing here but needs confirmation -- I probably prefer not making a change here since we don't have an observable benefit (the resolved URL is only used to resolve the container here) and I recall this being finicky.

Reply:
Actually just saw Helio is exporting this in #4014, so maybe would be good to add the containerRelativePathing actually.

Reply:
I would suggest writing tests even if they are very basic and just test the url resolver for now. We can keep adding more tests later as and when required once we have some basic tests.
As the usages of this increase, so does the changes to things going wrong.